x-pack/plugins/apm/public/components/shared/charts/CustomPlot/getEmptySeries.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { memoize } from 'lodash';
+import d3 from 'd3';
+
+export const getEmptySerie = memoize(
```

Comment:
is this spelling intentional? i might do `getEmptySeries` to match the file name

Reply:
Nah, that's just my broken English :p
Will fix.